quast/5.2.0/Dockerfile

```diff
@@ -0,0 +1,110 @@
+ARG QUAST_VER="5.2.0"
+
+## Builder ##
+FROM ubuntu:focal as builder
```

Comment:
I was wondering why focal was being used (the last Dockerfile used xenial), then I saw the python2 and other older dependencies. Does using jammy break things? (I'm assuming yes, but I figured I should ask)